articles/azure/azs-how-create-S2S-VPN.md

```diff
@@ -0,0 +1,222 @@
+---
+title: How to create a Site-to-Site connection in Azure Stack | UKCloud Ltd
+description: Create a S2S VPN gateway connection from an on-premises network to UKCLoud for Microsoft Azure 
+services: azure-stack
+author: Bailey Lawson
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create a Site-to-Site connection in Azure Stack
+toc_fullpath: Users/How To/azs-how-create-S2S-VPN.md
+toc_mdlink: azs-how-create-S2S-VPN.md
+---
+
+# How to create a Site-to-Site connection in Azure Stack
+
+## Overview
+
+This guide shows you how to use the UKCloud Azure Stack portal to create a Site-to-Site VPN from your on-premises network to your UKCloud for Microsoft Azure environment. 
+
+### Intended audience
+
+To complete the steps in this guide, you must have appropriate access to a subscription in the UKCloud Azure Stack portal.
+
+## Create a virtual network
+
+First, a virtual network will need to be created. This virtual network will be able to send and receive traffic through the virtual network gateway. A virtual network can not be associated with more than one gateway.
+
+1. Log in to the UKCloud Azure Stack portal.
+
+   For more detailed instructions, see the [*Getting Started Guide for UKCloud for Microsoft Azure*](azs-gs.md)
+
+2. In the favourites panel, select **Create a resource**.
+
+   ![Create a resource option in favourites panel](images/azsp_newmenu.png)
+
+3. In the **New** blade, select **Networking**.
+
+   ![Networking option in New blade](images/azs-browser-new-networking.png)
+
+4. In the **Featured** section, select **Virtual network**.
+
+   ![Create new networking resource](images/azs-browser-networking-create.png)
+
+5. In the **Create virtual network** blade, enter the following information:
+
+   **Name** - The name of the virtual network.
+
+   **Address Space** - The virtual network's address range in CIDR notation.
+
+   **Subscription** - This is your UKCloud for Microsoft Azure subscription.
+
+   **Resource Group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+   **Location** - This will be "frn00006", which is the location of the Azure Stack.
+
+   **Subnet Name** - The name of the first subnet within the virtual network.
+
+   **Address Range** - The subnet's address range in CIDR notation (e.g. 192.168.1.0). It must be contained by the address space of the virtual network. The address range of a subnet which is in use can't be edited.
+
+   ![Create new virtual network](images/azs-browser-create-virtual-network.png)

```

Comment:
Is it OK to show the address range in the screenshot, or does it need blurring out?

Reply:
Address range is fine as the network doesn't actually exist